src/components/PriorityOption/PriorityOption.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {
+    useState
+} from 'react';
+
+import PriorityIcon from '../PriorityIcon/PriorityIcon';
+
+
+import './PriorityOption.css';
+
+import {
+    options
+} from '../../constants/constants';
+
+function PriorityOption({ setTodoPriority }) {
+
+    const [selectedOption, setSelectedOption] = useState(0);
+    const [isOpenList, SetIsOpenList] = useState(false);
+
+    const handleClick = (value) => {
+        setTodoPriority(value);
+        setSelectedOption(value);
+        SetIsOpenList(!isOpenList);
+        // если я использую в setTodoPriority value, зачем мне стейт [selectedOption, setSelectedOption]
+    };
+
+    return (
+        <div className='priority' >
+            <div className={`selected-option ${isOpenList ? 'hidden-list' : ''}`}>
+                <div
+                    key={selectedOption}
+                    className={`priority-option-container priority-option-container${selectedOption}`}
+                    onClick={() => handleClick(selectedOption)}
+                    data-value={selectedOption}
+                >
+                    <PriorityIcon
+                        value={selectedOption}
+                        label={options[selectedOption].label}
+                        className={`option-value option-value${selectedOption}`}
+                    />
+                    <PriorityIcon
+                        value={selectedOption}
+                        label={options[selectedOption].label}
+                        className={`option-value option-value${selectedOption}`}
+                    />
+                    <PriorityIcon
+                        value={selectedOption}
+                        label={options[selectedOption].label}
+                        className={`option-value option-value${selectedOption}`}
+                    />
```

Comment:
Why didn't you mov all 3 icons to single components?
Do you use just one `<PriorityIcon` somewhere?